### DISCUSSION TITLE
Mirabella genio Wi-Fi Dimmable 19W UFO LED

### DISCUSSION THREAD

**Discussion by dunnycan** [milo526/homebridge-tuya-web#269] [Q&A]
Hi,
Just wondering if anyone has been able to get the Mirabella genie Wi-Fi Dimmable 19W UFO LED (from Kmart) to work with the Homebridge-Tuya-Web plugin. It doesn't seem to add in HOOBS or show in Apple Home. I can see it in the TuyaSmart app. Other Tuya products such as the GridConnect bulbs that I have bought from Bunnings seem to work, but just not the the UFO light. From what I've seen from other people online, other Mirabella Genio light bulb & products appear to work with this plugin. 

I note in the "How to check whether the API this library uses can control your device?", it gives instructions to copy a script to your PC with python installed or to https://repl.it/. I'm not quite sure how to do this because everything is set up through hoops. Can this be done on a Mac? If so are there any instructions to show to do this.
Thanks in advance for any help

**Comment by milo526** [comment]
Use https://repl.it it is an online python interpreter.
You can copy the script to that website and run the code, at the top, it states a few things you will have to fill in such as your email and password.
Without this information, I can not help you solve your problem.
